src/pages/scheduler/components/SchedulerCalendar.tsx

```diff
@@ -283,7 +285,7 @@ export function SchedulerCalendar({ calendarEvents }: SchedulerCalendarProps): J
       views={['work_week']}
       date={selectedDate}
       eventPropGetter={eventStyleGetter}
-      slotPropGetter={slotPropGetter}
+      slotPropGetter={(date) => slotPropGetter(date, isDarkMode)}
```

Comment:
Might be good to make a `handleSlotPropGetter` and inject the `isDarkMode` within that.